tests/check/signed_video_helpers.c

```diff
@@ -57,6 +57,8 @@ const char *axisDummyCertificateChain =
     "w3S8HMYSvMWbTCzN+qnq+GV1goSS6vjVr95EpDxCVIxkKOvuxhyVDg==\n"
     "-----END CERTIFICATE-----";
 
+const int64_t g_testTimestamp = 42;
```

Comment:
The prefix 'g_' looks like a glib constant. Either use ´testTimestamp´ or ´kTestTimestamp´.